pkg/openfeature/provider.go

```diff
@@ -56,3 +56,8 @@ type NumberResolutionDetail struct {
 	Value float64
 	ResolutionDetail
 }
+
+// Metadata provides provider metadata
```

Comment:
Might want to change this comment to `Metadata provides provider name`

Reply:
Done